packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.js

```diff
@@ -35,6 +35,7 @@ const Search = ({handleSearchBarToggle, isSearchBarExpanded}) => {
         openOnFocus: true,
         autoselect: false,
         hint: false,
+        tabAutocomplete: false,
```

Comment:
This produces extra CSS classes, especially since this option is not fit for us.